src/main/resources/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerStep/configLoad.js

```diff
@@ -0,0 +1,13 @@
+function loadParams() {
+    var div = document.getElementById('params');
+    fetch('${descriptor.descriptorUrl}/parameters?job=' + encodeURIComponent(document.getElementById('${jobFieldId}').value) + '&amp;context=${descriptor.contextEncoded}').then((rsp) => {
```

Comment:
Hi, I have solved the Test issues (needed to mvn clean and then test and remove script tags). I need a bit of help in how to pass jelly values to javascript. Stapler adjuncts dont allow to pass such variables and it is causing null pointer exceptions in one test case because it cannot find the HTML element

Reply:
I am not the expert, but I recall that the usual pattern is to [use `data-*` attributes](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes).

Reply:
Hi, thank you for being patient. The solution worked and the tests are passing now.